pkg/meta/model/job.go

```diff
@@ -837,11 +873,10 @@ type SubJob struct {
 	State        JobState        `json:"state"`
 	RowCount     int64           `json:"row_count"`
 	Warning      *terror.Error   `json:"warning"`
-	CtxVars      []any           `json:"-"`
+	NeedReorg    bool            `json:"-"`
 	SchemaVer    int64           `json:"schema_version"`
 	ReorgTp      ReorgType       `json:"reorg_tp"`
 	ReorgStage   ReorgStage      `json:"reorg_stage"`
-	NeedAnalyze  bool            `json:"need_analyze"`
```

Comment:
Will the modification of `CtxVars` and  delete of `NeedAnalyze` make json unmarshal fail while updating tidb?